lnwire/features.go

```diff
@@ -642,31 +642,39 @@ func (fv *RawFeatureVector) sizeFunc() uint64 {
 // Record returns a TLV record that can be used to encode/decode raw feature
 // vectors. Note that the length of the feature vector is not included, because
 // it is covered by the TLV record's length field.
-func (fv *RawFeatureVector) Record(recordType tlv.Type) tlv.Record {
+func (fv *RawFeatureVector) Record() tlv.Record {
 	return tlv.MakeDynamicRecord(
-		recordType, fv, fv.sizeFunc, rawFeatureEncoder,
```

Comment:
What about other instances that maybe were relying on this type? We should make sure to update those to avoid unintended breakage during deployment.